concurrency/runner_test.go

```diff
@@ -72,6 +72,99 @@ func TestForEachUser_ShouldReturnImmediatelyOnNoUsersProvided(t *testing.T) {
 	}))
 }
 
+func TestForEachJob(t *testing.T) {
+	var (
+		ctx = context.Background()
+	)
```

Comment:
Nit: let's just pass `context.Background()` to `ForEachJob`, it's not used anywhere else. Same in other tests where this applies.

Reply:
Sure, applied in https://github.com/grafana/dskit/pull/113/commits/a19cdadbd6970127e21bc6303e7f3b44d8a50fb9